dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.jsonbImpl-2.0.0.feature

```diff
@@ -0,0 +1,14 @@
+# This private impl feature corresponds to JSON-B 2.0 with the Yasson implementation
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.jsonbImpl-2.0.0
+singleton=true
+visibility=private
+-features=com.ibm.websphere.appserver.jsonp-2.0, \
+  com.ibm.websphere.appserver.classloading-1.0, \
+  com.ibm.websphere.appserver.jakarta.cdi-3.0,\
+  com.ibm.websphere.appserver.javaeeCompatible-8.0
+-bundles=com.ibm.ws.org.eclipse.yasson.1.0.jakarta, \
```

Comment:
The version of Yasson that will support EE9 will be Yasson 2.0, so we can name the bundle accordingly: `com.ibm.ws.org.eclipse.yasson.2.0`

Reply:
The yasson.1.0.jakarta bundle is the jakartaee-enabled bundle generated by the yasson.1.0/bnd.bnd file, below.  We'll revisit this.  If we can obtain a jakarta release of yasson, we'll add it to CNF, create a v2.0 bundle project, and modify this feature dependency accordingly.

Reply:
We'll do this when the jakarta yasson release candidate publishes.